extension/src/documents.ts

```diff
@@ -261,39 +413,37 @@ export class DocumentManager{
 		this.initialize();
 
 		AutoLispExt.Subscriptions.push(vscode.workspace.onDidCloseTextDocument((e: vscode.TextDocument) => {
-			if (this._opened.has(e.uri.fsPath) === true) {
-				this._opened.delete(e.uri.fsPath);
-			}
+			const source = this._cached.get(e.fileName.replace(/\//g, '\\'));
```

Comment:
e.fileName.replace(/\//g, '\\') also calls several times, maybe extract to a util small function

Reply:
I will change these to some variant of:
`this._cached.get(this.normalizePath(e.fileName))`